### ISSUE TITLE
install frontend dependencies: some vulnerability

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#1156]
At present

When I install tracim dependencies I can see many package with some vulnerability.

Expected

- [x] clean install frontend dependencies.

- [x] update dependencies if its necessary

**Comment by tracim** [comment]
The first task is to upgrade version of the dependencies.

**Comment by Skylsmoi** [comment]
upgrading every dependencies is mandatory but will require some time.

There is react that needs to be upgraded, webpack and babel. Theses are important upgrade (especially Babel) that all require a migration process that might be a bit complicated.

I'd say this will require 2 to 3 days

**Comment by Skylsmoi** [comment]
related https://github.com/tracim/tracim/issues/765

**Comment by Skylsmoi** [comment]
Babel has been upgraded properly to v7+
Webpack has been upgraded to webpack 4 too with a regression:
In v3, I was bundling tracim_frontend_lib with 2 entry points, one for the js app and one only for the css (stylus). This was really helpful to not repeat the css of tracim_frontend_lib with every apps.

Webpack was removing any css from the app's bundle since it was on the second entry point. This was done by the common Chunks Plugin.

Common chunks plugin has been removed from v4 with the addition of splitChunks

With webpack 4 and it's splitChunks, I couldn't find a way to reproduce this behavior. If I create 2 entry points like before, css is not put out of the first entry (so it's still repeated with every apps) and with splitChunks, I couldn't create a standalone bundle for the css only.

The branch is still WIP.

I have found a pretty ugly solution that works though:
The solution is to use 2 entry points like before (one for the app and one for the css only) and removes any `require('./MyFile.styl')` from components.
That way, components are bundled without any css, so it wont be repeated with every apps, and css gets bundled in it's own file.
It works but it's not ideal to remove the stylus files from components.
